ckan/tests/lib/test_app_globals.py

```diff
@@ -0,0 +1,17 @@
+from ckan.lib.app_globals import app_globals as g
+
+
+class TestGlobals(object):
+    def test_config_not_set(self):
+        # ckan.site_about has not been configured.
+        # Behaviour has always been to return an empty string.
+        assert g.site_about == ''
+
+    def test_config_set_to_blank(self):
```

Comment:
Maybe we should use the change_config decorator to avoid relying on actual configuration values on `test-core.ini`?

```
@helpers.change_config('ckan.site_description', '')
def test_config_set_to_blank(self):
    ...
```